test/test_estimates_api.py

```diff
@@ -89,7 +89,7 @@ def test_create_and_retrieve_shipping_estimate(self):
             create_order=False,
         )
         self.assertEqual(estimate.data.order, None)
-        self.assertEqual(estimate.data.type, "shipping")
+        self.assertEqual(estimate.data.type, "ecommerce")
```

Comment:
We're now re-routing to the ecommerce endpoint: https://github.com/patch-technology/patch/pull/5588